lib/syntax_tools/src/erl_syntax.erl

```diff
@@ -2828,7 +2838,7 @@ compact_list(Node) ->
 					  copy_attrs(Node,
 						     Node1));
 			_ ->
-			    Node 
```

Comment:
Nitpick: unrelated whitespace change. Please remove the others like it as well.

Reply:
Do you mean all the random trailing spaces? They were removed by my editor, and I suspect that will be the case for most editors. So in order to reduce the next diff, I would like to keep the commit.

Reply:
Changes which are not related to what the patch claims to do, mask the real reason for a code line looking like it does. If I'm trying to find a bug, I want git blame to point to the last functional change, not a completely unrelated patch doing a cosmetic change. I guess that's why the [contributor's guide](https://github.com/erlang/otp/blob/master/CONTRIBUTING.md) asks you to not fix preexisting whitespace errors in otherwise untouched source lines.

Reply:
I removed the commit that removes the trailing whitespace, but for posterity I'd like to mention the `--ignore-rev`/`--ignore-revs-file` flags to `git blame` which allows you to ignore/automatically re-blame.